### ISSUE TITLE
Improve contrast on certain colours used

### ISSUE THREAD

**Issue by lil5** [CollActionteam/clothing-loop#242]
The button "find a loop" has a contrast of 1.56 which fails WCAG.
https://coolors.co/contrast-checker/f7c86f-ffffff

Contrast on "Get ready to swap!" is at 2.5
https://coolors.co/contrast-checker/c58c41-fceed3

These are just quickly what I found on the Home page using Firefox's Accessibility Tools

**Comment by lil5** [comment]
```ts
const bronze = "#C58C41";
```

Bronze is used for more legible text and thus should have at least an AA rating.
I propose moving to `#A3610A` 
https://coolors.co/contrast-checker/a3610a-ffffff

**Comment by lil5** [comment]
moving to trello board
